databroker/src/main.rs

```diff
@@ -429,10 +429,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     #[cfg(feature = "viss")]
     {
+        let viss_bind_addr = args.get_one::<String>("viss-address").unwrap().parse()?;
+
         let viss_port = args
             .get_one::<u16>("viss-port")
             .expect("port should be a number");
-        let viss_addr = std::net::SocketAddr::new(ip_addr, *viss_port);
+        let viss_addr = std::net::SocketAddr::new(viss_bind_addr, *viss_port);
```

Comment:
This should be same as ip_addr only port different?

Reply:
No, that was the bug before. There is a seperate command line parameter for the VISS bind adress, and it makes sense to  have that to bind VISS to another interface depending on the usecase. I.e. bind on a "southbound" Ethernet for GRPC based feeders and e.g. only binding "low(er) risk" VISS to e.g. an WiFi Interface serving mobile clients etc.

Reply:
If `--viss-address` is not specified at startup but `--address` is, then that should be used also for the VISS interface, right?

Or in other words, only if `--viss-address` is explicitly specified should it be used.

Reply:
"Not sure", so currently it isn't and before --viss-address was ignored completely. I could change the behavior so that address is "for all" *unless* viss-adress overrides it.

However, using that logic wouldn't it be more logical to also have an addtional parameter grpc-(or val-) address that overrides it address only for GRPC? Or maybe let --adress being the "shortcut" for viss-address and val-address?

Reply:
> Or maybe let --adress being the "shortcut" for viss-address and val-address?

I think that's what I'm suggesting?

The most common by far is to just use `--address`. That will set the address for both the GRPC and VISS service. Is that what you mean by shortcut?

And in order to support having those services listen to different addresses / interfaces, if `--viss-address` can be specified. That gives you full control to set both the address of GRPC service _and_ the address of the VISS service.